web/src/types/tiktok-downloader.model.ts

```diff
@@ -0,0 +1,59 @@
+/** Interface for TikTokMedia */
+export interface TikTokMedia {
+    /** The format id of the media */
+    format_id: string 
```

Comment:
Can you clarify why is this in snake_case?

Reply:
I forgot about it. No clarification required